VKApp/VKApp/Model/Group/GroupItem.swift

```diff
@@ -6,14 +6,20 @@ import RealmSwift
 /// GroupItem
 @objcMembers
 final class GroupItem: Object, Codable {
+    // MARK: - Public Property
+
     dynamic var id: Int
     dynamic var name: String
     dynamic var photo: String
```

Comment:
```suggestion
    /// id пользователя
    dynamic var id: Int
    /// имя пользователя
    dynamic var name: String
    /// путь, по которому  хранится фото пользователя
    dynamic var photo: String
```
мы же вчера на синке скидывали и в группу и обсуждали как оформляем модели